spec/models.py

```
@@ -0,0 +1,227 @@
+    current_progress: str = ""
+    completed_steps: List[str] = Field(default_factory=list)
+    remaining: List[str] = Field(default_factory=list)
+    always: List[str] = Field(default_factory=list)
+    ask_first: List[str] = Field(default_factory=list)
+    never: List[str] = Field(default_factory=list)
+    self_check_instructions: List[str] = Field(default_factory=list)
+    process_requirements: List[str] = Field(default_factory=list)
+
+
+class TemplateSpecContent(BaseModel):
+    """模板 Spec 内容模型（用于模板填充）"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)